#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("outlier_distance_threshold", double_t, 0, "threshold to rmeove outliers", 0.01, 0.0, 0.2)
gen.add("min_inliers", int_t, 0, "the minimum number on a edge", 10, 0, 1000)
gen.add("duplication_angle_threshold", double_t, 0, "angular threshold to regard two edges as same", 0.1, 0.0, 1.57)
gen.add("duplication_distance_threshold", double_t, 0, "distance threshold to regard two edges as same", 0.01, 0.0, 1.0)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "EdgeDepthRefinement"))
